src/app/authentication/verification/verification.component.html

```diff
@@ -1,9 +1,9 @@
 <form  #loginForm="ngForm" >
   <img src="https://www.google-melange.com/archive/gci/2014/orgs/mifos/logo-200.png" alt="No Image found">
-    <mat-form-field>
+    <mat-form-field class="font-style ">
```

Comment:
Please remove space in the class

Reply:
okay sure

Reply:
Hi Gaurav, as you have suggested. I have indented all the code.